src/SFA.DAS.Forecasting.Domain.UnitTests/Estimations/AccountEstimationProjectionTests.cs

```diff
@@ -266,7 +266,7 @@ public void Then_The_Estimation_Uses_Actual_And_Modelled_Costs_Against_My_Transf
             //Assert
             var actual = _estimationProjection.Projections.OrderBy(c => c.Year).ThenBy(c => c.Month).FirstOrDefault();
             Assert.IsNotNull(actual);
-            Assert.AreEqual(9940 , actual.AvailableTransferFundsBalance);
+            Assert.AreEqual(9990 , actual.AvailableTransferFundsBalance);
```

Comment:
Can we fix the test data so that we don't have to change the expected values of the test every time we commit (how do we know if the code has actually been broken)

Reply:
TestData changed to use the Current year